content/news/core-monthly/2023/core-monthly-2023-07-01-2023-07-31.md

```diff
@@ -0,0 +1,530 @@
+
+We also noticed that on some servers upgrade fails if you perform it on PHP 8.1. We will investigate this issue and try to fix it in the next patch version. For the time being, if you encounter some problems, it's recommended to try on PHP 7.4 instead of PHP 8.1. You can change the PHP version after the upgrade.
+
+When you read this article, the 8.1.1 version should have already been released.
+
+### PrestaShop 9
+
+A lot of work has been done on PrestaShop 9 in July. If you see a list of merged PRs on the `develop` branch you will notice a lot of PRs like: "Add/Replace something as/with a Twig component.", this is an ongoing project that aims to rework back office to be fully Symfony-based. At the moment, full Symfony layout is behind the feature flag. 
+
+Other than that, a lot of cleaning! Removed deprecations, unused services. Also [reworked mechanism of dealing with Feature Flags](https://github.com/PrestaShop/PrestaShop/pull/32923) has been merged. You can learn more about this mechanism from the [@boherm](https://github.com/boherm)'s [presentation from Live Update](https://youtu.be/jzKBQM3fprY?t=2429).
+
+The project that also deserves attention is the one that Daniel [@Hlavtox](https://github.com/hlavtox) started, and which is all about improving performance of the front office. A few first pull requests have been merged already, and there are more to come. Examples: [improved loading of product features](https://github.com/PrestaShop/PrestaShop/pull/33335), [category controller using LazyArray mechanism](https://github.com/PrestaShop/PrestaShop/pull/32653). Daniel's goal is to make `Product::getProductProperties()` method lighter, and, eventually, replace it with a more performant solution.
+
+### Autoupgrade module
+
+Thanks to [@Hlavtox](https://github.com/Hlavtox) we managed to release a new version of the `autoupgrade` module with a few very important fixes. Version 4.16.2 introduces a few extra checks in the process, which should prevent some issues that were reported by users. There are also some improvements such as more informative log messages, and a fix for the servers with `opcache` enabled.
+
+### Changes in the developer documentation
+
+In July, we published a new guide explaining how to create a basic configuration form for your module. You can visit the [guide](https://devdocs.prestashop-project.org/8/modules/creation/adding-configuration-page-modern/) to learn more about it. There is also an [example module](https://github.com/PrestaShop/example-modules/tree/master/demosymfonyformsimple) that you can use as a reference.
+
+### Meet at PrestaShop Developer conference in November!
+
+If you didn't know yet, we are organizing a PrestaShop Developer conference in November! It will be a great opportunity to meet the PrestaShop team, learn about the latest news and best practices, and share your experience with other developers. The conference will take place in Paris, France, on November 30th.
```

Comment:
```suggestion
If you didn't know yet, we are organizing a PrestaShop Developer Conference in November! It will be a great opportunity to meet the PrestaShop team, learn about the latest news, best practices, and share your experience with other developers. The conference will take place in Paris, France, on November 30th.
```